plugins/source/azure/resources/services/security/regulatory_compliance_standards.go

```diff
@@ -30,9 +32,22 @@ func fetchRegulatoryComplianceStandards(ctx context.Context, meta schema.ClientM
 	for pager.More() {
 		p, err := pager.NextPage(ctx)
 		if err != nil {
-			return err
+			// check if we encountered err we can skip
+			var respErr *azcore.ResponseError
+			if !errors.As(err, &respErr) {
+				return err
+			}
+
+			const noStdPricingBundleErrorCode = `Subscription with no standard pricing bundle`
+			if respErr.ErrorCode != noStdPricingBundleErrorCode {
+				return err
+			}
+
+			cl.Logger().Warn().Err(err).Msg("skip fetch due to err from Azure")
```

Comment:
As a general approach we try to avoid classifying API errors and let users handle them.
What's the added value of logging this error in the plugin's code instead of returning it and letting the SDK do it?

It's seems that in order to solve this error users need to either use a different subscription or skip this table if not relevant for their subscription.

So in both cases we'd like to report it, WDYT?

Reply:
@erezrokah
Yeah, I just made a PR to discuss whether we want to skip this issue as err in our reporting or not.
Basically, it couldd only be. corrected by the user.
We can close the #7197 issue with wontfix resoluttion.